doc-content/enterprise-only/admin-and-config/deployment-descriptor-configuration-proc.adoc

```diff
@@ -104,4 +104,4 @@ The following table lists these along with the permissible and default values fo
 
 |===
 
-WARNING: Do not use the Singleton runtime strategy with the EJB Timer Scheduler (the default scheduler in {KIE_SERVER}) in a production environment. This combination can result in Hibernate problems under load. For more information about this limitation, see https://access.redhat.com/solutions/4065521[Hibernate issues with Singleton strategy and EJBTimerScheduler].
+WARNING: Do not use the Singleton runtime strategy with the EJB Timer Scheduler (the default scheduler in {KIE_SERVER}) in a production environment. This combination can result in Hibernate problems under load. `PER_PROCESS_INSTANCE` is recommended for runtime strategy if there is no specific reason to use other strategies. For more information about this limitation, see https://access.redhat.com/solutions/4065521[Hibernate issues with Singleton strategy and EJBTimerScheduler].
```

Comment:
I would just rephrase it a little bit to `` `PER_PROCESS_INSTANCE` is the recommended runtime strategy...``

Reply:
@MarianMacik, thanks!
I have updated the sentence. Please review